core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/LeonardoClient.scala

```diff
@@ -0,0 +1,25 @@
+package org.broadinstitute.dsde.rawls.workspace
+
+import org.broadinstitute.dsde.workbench.client.leonardo.api.AppsV2Api;
+import org.broadinstitute.dsde.workbench.client.leonardo.model.CreateAppRequest;
+import org.broadinstitute.dsde.workbench.client.leonardo.model.AppType;
+
+import scala.concurrent.Future
+import org.broadinstitute.dsde.workbench.client.leonardo.ApiClient
+
+class LeonardoClient(val leonardoBasePath: String) {
+
+  def getAppsV2leonardoApi(accessToken: String): AppsV2Api = {
+    val apiClient = new ApiClient()
+    apiClient.setAccessToken(accessToken)
+    apiClient.setBasePath(leonardoBasePath)
+    new AppsV2Api(apiClient)
+  }
+
+  def createWDSInstance(token: String, workspaceId: String, appName: String): Unit = {
+    val createAppRequest = new CreateAppRequest();
```

Comment:
Scala does not require trailing `;`, and it is idiomatic to omit those. Check the whole PR for this!

Reply:
Removed